.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
```

Can you please also add the following here?

```suggestion
    labels:
      - dependencies
    schedule:
```

And include a new `dependencies` section in `.hashibot.hcl` in the `pull_request_path_labeler` behavior?

```hcl
behavior "pull_request_path_labeler" "service_labels" {
  label_map = {
    # label provider related changes
    "provider" = [
      # ... existing
    ]
    "dependencies" = [
      ".github/dependabot.yml",
      "renovate.json",
    ]
    "documentation" = [
```

Thanks!

---

Oh ha, reading the documentation more, `dependencies` is already set by default: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#labels

> By default, GitHub Dependabot raises all pull requests with the `dependencies` label

So ignore that -- but I think we'd like to get the hashibot change in there. 👍